iron-a11y-keys-behavior.html

```diff
@@ -102,6 +109,8 @@
         var lKey = key.toLowerCase();
         if (lKey === ' ' || SPACE_KEY.test(lKey)) {
           validKey = 'space';
+        } else if (ESC_KEY.test(lKey)) {
```

Comment:
this code is basically equivalent to `lKey === 'escape'`, right?

did you mean to add an `/i` after the regex, as it seems like the linked URL actually says `Escape` (uppercase E)?